packages/e2e/features/ui/components/authenticator/forgot-password.feature

```diff
@@ -76,3 +76,18 @@ Feature: Reset Password
     Then I see "Password must have lower case letters"
     Then I see "Password must have at least 8 characters"
     Then I confirm "Password must have numbers" error is accessible in new password field
+
+  @react @vue @angular @react-native
```

Comment:
Did we test that this test runs successfully for RN? I don't see `I see {string} is not disabled` step defined for Detox. Currently RN e2es are disabled in CI so we should run them locally before adding or modifying any that may fail in CI at a later date when we re-enable RN e2es.